packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -503,6 +503,18 @@ describe('<Autocomplete />', () => {
       });
     });
 
+    it('should apply the multiple class', () => {
```

Comment:
Could you also include a separate test for overriding via theme using `styleOverrides` for `multiple`?